package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cortexapps/backstage-plugin",
-  "version": "2.11.7",
+  "version": "2.11.8",
```

Comment:
There was merge of another PR by Dušan. Please update version to `2.12.2` and resolve conflicts.